### PR TITLE
Replace Travis with github actions

### PR DESCRIPTION
**Pull request by carrascomj**
Use github actions for CI/CD instead of travis. This PR makes cameo's CI consistent with the cobrapy ecosystem and it avoids some issues with travis.

This PR tries to replicate the previous setup with travis. Further improvements should be explored in future PRs (black, isort, python/os versions...).

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        sudo apt install openbabel
+        python -m pip install --upgrade pip setuptools wheel
+        python -m pip install tox tox-gh-actions
```
Comment (carrascomj) [inline .github/workflows/main.yml:42]: Py3.6 `.travis/install_cplex.sh`:
- Mock and check that secret is not showed in log.
- Adapt it to python3.8.

**Review by carrascomj — COMMENTED** [review]
Required changes before merging.

File: .github/workflows/main.yml (hunk 1 above)
Comment (carrascomj) [reply .github/workflows/main.yml:37]: CPLEX path and unpacking must be corrected.

**Review by phantomas1234 — APPROVED** [review]
LGTM, thanks!

**Pull request merged**